l2geth/rollup/sync_service.go

```diff
@@ -31,6 +31,7 @@ var errShortRemoteTip = errors.New("Unexpected remote less than tip")
 // L2GasPrice slot refers to the storage slot that the execution price is stored
 // in the L2 predeploy contract, the GasPriceOracle
 var l2GasPriceSlot = common.BigToHash(big.NewInt(1))
+var l2GasPriceOracleAddress = common.HexToAddress("0x420000000000000000000000000000000000000F")
```

Comment:
This is the default value?

Reply:
Yes this is the exact value, see https://github.com/ethereum-optimism/optimism/blob/8fee7bed6b63a861fa3e0300869d544bc3a75b45/packages/contracts/src/predeploys.ts#L21